modules/client/src/interfaces.ts

```diff
@@ -1,87 +1,12 @@
 // This file contains the definitions of the general purpose DAO client
 
 import {
-  DaoAction,
-  DaoRole,
-  GasFeeEstimation,
-  IInterfaceParams,
   IPagination,
   IPluginInstallItem,
 } from "./client-common/interfaces/common";
 import { keccak256 } from "@ethersproject/keccak256";
 import { toUtf8Bytes } from "@ethersproject/strings";
 import { BigNumber } from "@ethersproject/bignumber";
-import { IClientCore } from "./client-common/interfaces/core";
-/** Defines the shape of the general purpose Client class */
-export interface IClientMethods extends IClientCore {
-  create: (params: ICreateParams) => AsyncGenerator<DaoCreationStepValue>;
-  /** Retrieves the asset balances of the given DAO, by default, ETH, DAI, USDC and USDT on Mainnet*/
-  getBalances: (
-    daoAddressOrEns: string,
-    tokenAddresses: string[],
-  ) => Promise<AssetBalance[] | null>;
-  /** Retrieves the list of transfers from or to the given DAO, by default, ETH, DAI, USDC and USDT on Mainnet*/
```

Comment:
I guess we remove all of this because the client now relates to the class itself?

Reply:
yes